vendor-extra/JatsContentBundle/src/EventListener/BuildView/IsoDateListener.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\EventListener\BuildView;
+
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\SimplifiedViewConverterListener;
+use Libero\ViewsBundle\Views\View;
+use function is_string;
+use function Libero\LiberoPageBundle\is_valid_date;
+use function Libero\ViewsBundle\array_has_key;
+use function preg_match;
+
+final class IsoDateListener
+{
+    private const DATE_PATTERN = '~^([0-9]{4}-[0-9]{2}-[0-9]{2})(?:$|T)~';
+
+    use SimplifiedViewConverterListener;
+
+    protected function handle(Element $object, View $view) : View
+    {
+        $date = $object->getAttribute('iso-8601-date');
+
+        if (!is_string($date)) {
```

Comment:
Would use `hasAttribute()` here, but PHPStan doesn't then recognise that `getAttribute()` will only return a string afterwards.